src/main/resources/assets/bettergrass/lang/zh_cn.json

```diff
@@ -0,0 +1,16 @@
+{
+    "bettergrass.title": "BetterGrassify （更好的草化）设置",
+    "bettergrass.betterGrassMode": "预设模式",

```

Comment:
Can you change this to something like **Better Grass Mode**, instead of "Preset Mode"?